app/src/main/java/com/pac_man/GameInitializer.java

```diff
@@ -0,0 +1,145 @@
+package com.pac_man;
+
+import com.bridge.processinputhandler.listeners.KeyboardListener;
+import com.bridge.renderHandler.sprite.Size;
+import com.bridge.renderHandler.sprite.Sprite;
+import com.bridge.renderHandler.sound.Sound;
+import com.bridge.renderHandler.sprite.Coord;
+import com.pac_man.Map.Maze;
+import com.pac_man.characters.Geometry.Position;
+import com.pac_man.characters.Ghost.*;
+import com.pac_man.characters.Ghost.BlinkyChaser;
+import com.pac_man.characters.Ghost.ClydeChaser;
+import com.pac_man.characters.Ghost.InkyChaser;
+import com.pac_man.characters.Ghost.PinkyChaser;
+import com.pac_man.characters.Pacman.Pacman;
+
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameInitializer {
+    public void init() {
+        URL pacmanImg = getClass().getResource("app/src/main/java/com/pac_man/Resources/PacMan/pr1.png");
```

Comment:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.net.URL.toURI()" because "pacmanImg" is null
	at com.pac_man.GameInitializer.init(GameInitializer.java:30)
	at com.pac_man.App.main(App.java:13)
	
	The project its not compiling, solve this issue and others similar related to obtain the image.